### DISCUSSION TITLE
SSD (not nvme)

### DISCUSSION THREAD

**Discussion by jigsurfer** [Joshua-Riek/ubuntu-rockchip#557] [General]
Hi,

I want to take use of my spare SSD for the on board nvme slot, but its seem not recognize the ssd, only nvme. 
It does recognize if i put ssd + external dock using USB port.

I did follow instruction on orange pi 5. My hardware is orangpi 5 plus.

Could you please help me to know how to let the hardware working foe the built in nvme.

Thank you

**Comment by hakotani-o** [comment]
Hi, Does Orangi pi 5 plus support M.2 SSD?
There are hardware differences between Orange pi 5 and Orange pi 5 plus.

**Reply by bagong** [reply]
Yes, very much so. It has 2 M.2 slots, the one at the bottom supports pcie-3 4-lane and there is enough place for a full length stick. Speeds are as hoped, "Disks" reports 3MB/s read speed in the default test.

**Reply by jigsurfer** [reply]
Yes it does. OPi Os config options also show 2 options for
 ssd-sata0 & ssd-sata2.

It mean its possible to take use of both pcie slots as memory sticks on same syst board as SSD if you dont use other one for netword card.

I found that the entire issue relate ssd cnfuguration has bg trouble. Nvme is recognized but never Sata nomatter what solutions you re trying.
